pytest_tests/testsuites/object/test_object_api.py

```diff
@@ -337,6 +341,69 @@ def test_object_search_should_return_tombstone_items(
                     object_type == "TOMBSTONE"
                 ), f"Object wasn't deleted properly. Found object {tombstone_oid} with type {object_type}"
 
+    @allure.title("Validate objects search by common prefix")
+    def test_search_object_api_common_prefix(self, default_wallet: str, simple_object_size: int):
+        FILEPATH_ATTR_NAME = "FilePath"
+        cli = NeofsCli(self.shell, NEOFS_CLI_EXEC, WALLET_CONFIG)
+        wallet = default_wallet
+        cid = create_container(wallet, shell=self.shell, endpoint=self.cluster.default_rpc_endpoint)
+
+        objects = {}
+        for _ in range(5):
```

Comment:
Looks like a magic number

Reply:
It is definitely a magic number that is used inside this test in one place. I can replace it with a string but it would be just one more line without a reason right now.

Reply:
> It is definitely a magic number that is used inside this test in one place. I can replace it with a string but it would be just one more line without a reason right now.

For debugging in the future, this would be a good reference

Reply:
fixed